### PR TITLE
fix(slider): thumb no longer appears above track at min

### PR DESCRIPTION
**Pull request by anveshmekala**
**Related Issue:** #5382 

## Summary

This will fix the thumb position to be on the slider track at min value.

After the fix :
EDA939A0-5D0C-4976-A034-DDD9442BF380_4_5005_c

### CODE REVIEW

File: src/components/slider/slider.tsx [hunk 1]
```diff
@@ -1428,16 +1428,14 @@ export class Slider
...
    */
-  private determineGroupSeparator = (value): string => {
+  private determineGroupSeparator = (value: number): number | string => {
     const lang = this.globalAttributes["lang"] || document.documentElement.lang || "en";
-    if (value) {
-      return this.groupSeparator
-        ? localizeNumberString(value.toString(), lang, this.groupSeparator, this.numberingSystem)
-        : value;
-    }
+    return this.groupSeparator && value
```
Comment (driskull) [inline src/components/slider/slider.tsx:1437]: What if `value` is `0`? Should it localize it still?

Comment (anveshmekala) [reply src/components/slider/slider.tsx:1437]: It should. Updated the code.

Comment (anveshmekala) [reply src/components/slider/slider.tsx:1437]: And also, earlier it is required to parse `group-separator` boolean to localize the numbers in `slider`. With this fix `group-separator `is not required. 

cc: @benelan , @geospatialem

**Review by driskull — APPROVED** [review]
👍 🎉

**Pull request merged**